Test test31_invalidFiles() failed: Error thrown
java.lang.AssertionError: Error thrown
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.run(WalkUtil.java:177)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.check(WalkUtil.java:135)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.check(WalkUtil.java:125)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.check(WalkTest.java:237)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest$TestCase.test(WalkTest.java:267)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest$TestCase.test(WalkTest.java:256)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.test31_invalidFiles(WalkTest.java:86)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.security.AccessControlException: access denied ("java.io.FilePermission" "/..//..." "read")
at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:488)
at java.base/java.security.AccessController.checkPermission(AccessController.java:1071)
at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:411)
at java.base/java.lang.SecurityManager.checkRead(SecurityManager.java:742)
at java.base/java.io.File.exists(File.java:831)
at info.kgeorgiy.ja.kupriyanov.walk.Walk.calculateHash(Walk.java:61)
at info.kgeorgiy.ja.kupriyanov.walk.Walk.main(Walk.java:48)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.run(WalkUtil.java:173)
... 9 more
Test test50_whitespaceSupport() failed: Wrong hash for __Test__Walk__/test50_whitespaceSupport()/  __  __ _ _    _ __   ___     ==> expected: <396d3551> but was: <00000000>
org.opentest4j.AssertionFailedError: Wrong hash for __Test__Walk__/test50_whitespaceSupport()/  __  __ _ _    _ __   ___     ==> expected: <396d3551> but was: <00000000>
at org.junit.jupiter.api@5.10.2/org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
at org.junit.jupiter.api@5.10.2/org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
at org.junit.jupiter.api@5.10.2/org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
at org.junit.jupiter.api@5.10.2/org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
at org.junit.jupiter.api@5.10.2/org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1156)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.checkOutput(WalkUtil.java:155)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.check(WalkUtil.java:136)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.check(WalkUtil.java:125)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.check(WalkTest.java:237)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest$TestCase.test(WalkTest.java:267)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest$TestCase.test(WalkTest.java:256)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.testRandomFiles(WalkTest.java:204)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.testAlphabet(WalkTest.java:214)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.test50_whitespaceSupport(WalkTest.java:120)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Test test61_invalidInput() failed: Error thrown
java.lang.AssertionError: Error thrown
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.run(WalkUtil.java:177)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.test61_invalidInput(WalkTest.java:156)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.security.AccessControlException: access denied ("java.io.FilePermission" "*" "read")
at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:488)
at java.base/java.security.AccessController.checkPermission(AccessController.java:1071)
at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:411)
at java.base/java.lang.SecurityManager.checkRead(SecurityManager.java:742)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:141)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:106)
at java.base/java.io.FileReader.<init>(FileReader.java:103)
at info.kgeorgiy.ja.kupriyanov.walk.Walk.main(Walk.java:39)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.run(WalkUtil.java:173)
... 4 more
Test test62_invalidOutput() failed: Error thrown
java.lang.AssertionError: Error thrown
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.run(WalkUtil.java:177)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.test62_invalidOutput(WalkTest.java:163)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.security.AccessControlException: access denied ("java.io.FilePermission" "*" "write")
at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:488)
at java.base/java.security.AccessController.checkPermission(AccessController.java:1071)
at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:411)
at java.base/java.lang.SecurityManager.checkWrite(SecurityManager.java:833)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:218)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:118)
at java.base/java.io.FileWriter.<init>(FileWriter.java:144)
at info.kgeorgiy.ja.kupriyanov.walk.Walk.main(Walk.java:45)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.run(WalkUtil.java:173)
... 4 more
Test test63_invalidFiles() failed: Error thrown
java.lang.AssertionError: Error thrown
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.run(WalkUtil.java:177)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.check(WalkUtil.java:135)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.check(WalkUtil.java:125)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.check(WalkTest.java:237)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest$TestCase.test(WalkTest.java:267)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest$TestCase.test(WalkTest.java:256)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.testRandomFiles(WalkTest.java:204)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.testAlphabet(WalkTest.java:214)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkTest.test63_invalidFiles(WalkTest.java:170)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.security.AccessControlException: access denied ("java.io.FilePermission" "__Test__Walk__/test63_invalidFiles()/\\**\***\\*\\***\**\\\" "read")
at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:488)
at java.base/java.security.AccessController.checkPermission(AccessController.java:1071)
at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:411)
at java.base/java.lang.SecurityManager.checkRead(SecurityManager.java:742)
at java.base/java.io.File.exists(File.java:831)
at info.kgeorgiy.ja.kupriyanov.walk.Walk.calculateHash(Walk.java:61)
at info.kgeorgiy.ja.kupriyanov.walk.Walk.main(Walk.java:48)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at info.kgeorgiy.java.advanced.walk/info.kgeorgiy.java.advanced.walk.WalkUtil.run(WalkUtil.java:173)
... 11 more
ERROR: Tests: failed
